ray/tests/common.py

```diff
@@ -0,0 +1,269 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+from datadog_checks.dev import get_docker_hostname, get_here
+
+HERE = get_here()
+
+RAY_VERSION = os.environ.get("RAY_VERSION")
+
+SERVE_PORT = "8000"
+HEAD_METRICS_PORT = "8080"
+WORKER1_METRICS_PORT = "8081"
+WORKER2_METRICS_PORT = "8082"
+WORKER3_METRICS_PORT = "8083"
+HEAD_DASHBOARD_PORT = "8265"
+
+HEAD_OPENMETRICS_ENDPOINT = f"http://{get_docker_hostname()}:{HEAD_METRICS_PORT}"
+WORKER1_OPENMETRICS_ENDPOINT = f"http://{get_docker_hostname()}:{WORKER1_METRICS_PORT}"
+WORKER2_OPENMETRICS_ENDPOINT = f"http://{get_docker_hostname()}:{WORKER2_METRICS_PORT}"
+WORKER3_OPENMETRICS_ENDPOINT = f"http://{get_docker_hostname()}:{WORKER3_METRICS_PORT}"
+SERVE_URL = f"http://{get_docker_hostname()}:{SERVE_PORT}"
+
+HEAD_INSTANCE = {
+    "openmetrics_endpoint": HEAD_OPENMETRICS_ENDPOINT,
+}
+
+WORKER1_INSTANCE = {
+    "openmetrics_endpoint": WORKER1_OPENMETRICS_ENDPOINT,
+}
+
+WORKER2_INSTANCE = {
+    "openmetrics_endpoint": WORKER2_OPENMETRICS_ENDPOINT,
+}
+
+WORKER3_INSTANCE = {
+    "openmetrics_endpoint": WORKER3_OPENMETRICS_ENDPOINT,
+}
+
+MOCKED_HEAD_INSTANCE = {
+    "openmetrics_endpoint": "http://ray-head:8080",
+}
+
+MOCKED_WORKER_INSTANCE = {
+    "openmetrics_endpoint": "http://ray-worker:8081",
+}
+
+E2E_METADATA = {
+    'env_vars': {
+        'DD_LOGS_ENABLED': 'true',
+    },
+}
+
+HEAD_METRICS = [
```

Comment:
I think you can reduce meaningful duplication in the tests if you add the `ray.` prefix here, eg `HEAD_METRICS = ['ray.' + m for m in HEAD_METRICS]`

Reply:
good idea!